spec/konfa_spec.rb

```diff
@@ -369,4 +360,82 @@ def allowed_variables
       expect(MyOtherTestKonfa.get :my_var).to eq 'true'
     end
   end
+
+
+  context "#read_from" do
```

Comment:
We should add tests for multiple calls to read_from to verify that merging values will work as expected. This will serve well for regression purposes.

Reply:
True. I'll add those